.github/workflows/cla-check.yml

```diff
@@ -2,6 +2,7 @@
 name: Canonical Contributor Licence Agreement check
 
 on:
+  workflow_call:
```

Comment:
We don't claim ownership for this check, so there probably isn't a need for this.

Not to say that this change does harm, but if an existing project with its own workflows is embedding the Starter Pack, they should have a CLA check as a main part of their project, not just docs.

Unless there are public projects at Canonical that _don't_ require a CLA check for the functional code?

@SecondSkoll @tang-mm Any objection to listing this workflow in [this section of the tutorial](https://canonical-starter-pack.readthedocs-hosted.com/stable/tutorial/set-up/#remove-the-unneeded-files)?

Reply:
You have a point. I'm undoing this change